include/deal.II/multigrid/mg_transfer_internal.h

```diff
@@ -38,6 +40,76 @@ namespace internal
                            std::vector<std::vector<std::pair<types::global_dof_index, types::global_dof_index> > > &copy_indices,
                            std::vector<std::vector<std::pair<types::global_dof_index, types::global_dof_index> > > &copy_indices_global_mine,
                            std::vector<std::vector<std::pair<types::global_dof_index, types::global_dof_index> > > &copy_indices_level_mine);
+
+
+
+    /**
+     * Given the collection of child cells in lexicographic ordering as seen
+     * from the parent, this function computes the first index of the given
+     * child
+     */
+    template <int dim>
+    unsigned int
+    compute_shift_within_children(const unsigned int child,
```

Comment:
I have no idea what this function does, or what the documentation describes, but I guess it will make sense to those who use the function internally :-)